### PR TITLE
Feature/multiple rules files

### PR DESCRIPTION
**Pull request by zipkid**
Allow the use of multiple alert rules files.

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -158,6 +158,7 @@
   Array $remote_read_configs  = $::prometheus::params::remote_read_configs,
   Array $remote_write_configs = $::prometheus::params::remote_write_configs,
   $alerts                     = $::prometheus::params::alerts,
+  $extra_alerts               = {},
```
Comment (bastelfreak) [inline manifests/init.pp]: Please add a datatype for this new parmeter and documentation in the header.

Comment (zipkid) [reply manifests/init.pp]: Ok, i will divert from the existing style :-)

File: manifests/alerts.pp [hunk 2]
```diff
@@ -11,29 +11,32 @@
 #  [*alertfile_name*]
 #  Filename to use when storing the alerts file
 #
-class prometheus::alerts (
-  String $location,
+define prometheus::alerts (
```
Comment (bastelfreak) [inline manifests/alerts.pp]: can you please write a spec file with unit tests for this new defined resource?

File: manifests/alerts.pp [hunk 3]
```diff
@@ -11,29 +11,32 @@
...
   Variant[Array,Hash] $alerts,
-  String $alertfile_name = $prometheus::params::alertfile_name,
-) inherits prometheus::params {
+  String $location = "$::prometheus::config_dir/rules",
```
Comment (bastelfreak) [inline manifests/alerts.pp]: please adjust the documenation in the header

**Comment by bastelfreak** [comment]
Hi @zipkid , thanks for the PR. I added some inline comments.

**Comment by zipkid** [comment]
@bastelfreak Comments handled and spec test added.

File: manifests/alerts.pp [hunk 4]
```diff
@@ -1,39 +1,43 @@
...
 #
-class prometheus::alerts (
-  String $location,
+define prometheus::alerts (
   Variant[Array,Hash] $alerts,
-  String $alertfile_name = $prometheus::params::alertfile_name,
-) inherits prometheus::params {
-  if ( versioncmp($::prometheus::version, '2.0.0') < 0 ){
-    file { "${location}/${alertfile_name}":
+  String $location = "${::prometheus::config_dir}/rules",
```
Comment (bastelfreak) [inline manifests/alerts.pp:13]: Datatypes \o/

**Comment by bastelfreak** [comment]
This looks good to me. @tuxmea can you review this as well?

**Review by bastelfreak — APPROVED** [review]

**Comment by tuxmea** [comment]
Needs a major version bump as interface changes from class to define.

**Review by tuxmea — APPROVED** [review]

**Comment by zipkid** [comment]
I am willing to bump the version but unsure about the actions / steps to do it....

**Comment by bastelfreak** [comment]
@zipkid, don't worry. We will deal with this in our release workflow.

**Pull request merged**